src/map/routing.c

```diff
@@ -144,7 +150,7 @@ static inline void ordered_queue_reduce_index(int index, int offset, int dist)
     }
 }
 
-static void ordered_enqueue(int next_offset, int current_dist, int remaining_dist)
+static void ordered_enqueue(int offset, int next_offset, int current_dist, int remaining_dist)
```

Comment:
I don't think this change is needed as you change nothing else in the function.

Reply:
I was totally using it I swear. Then stuff happened and I didn't need it anymore :)